plugin/src/main/java/io/jenkins/plugins/remotingkafka/KafkaComputerLauncher.java

```diff
@@ -33,9 +40,17 @@
     @CheckForNull
     private transient volatile ExecutorService launcherExecutorService;
 
-    @DataBoundConstructor
-    public KafkaComputerLauncher() {
+    private String kafkaUsername;
```

Comment:
What is the need to duplicate the fields here? They do not seem to be used

Reply:
@oleg-nenashev this field is used for users to input username.

Reply:
Is it being used somewhere? I do not see it in the code

Reply:
@oleg-nenashev yes this is my mistake, this should be used for security setup when connecting consumer and producer clients to the Kafka. I will make my changes.

Reply:
@oleg-nenashev actually it was not a mistake `kafkaUsername` is used by `getKafkaUsername()`, which is called by the UI in `main.jelly`